source/code/plugin/out_mdm.rb

```diff
@@ -154,7 +144,7 @@ def get_access_token
             parsed_json = JSON.parse(token_response.body)
             @token_expiry_time = Time.now + @@tokenRefreshBackoffInterval * 60 # set the expiry time to be ~thirty minutes from current time
             @cached_access_token = parsed_json["access_token"]
-            @log.info "Successfully got access token"
```

Comment:
nit: formatting seems to be off.. could you pls fix it if possible?

Reply:
@rashmichandrashekar , this file reverted. looks like you reviewed before revert?, btw, i didnt see any issue with formatting.